### PR TITLE
Remove reference to sizeclasses.com

### PR DESCRIPTION
**Pull request by igorsantos07**
### Platforms affected
iOS

### Motivation and Context
That website is now parked, probably got abandoned.

### Description
Just removed a line referring to the defunct website.

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [x] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

**Comment by igorsantos07** [comment]
BTW, that checklist seems generic, since this docs repository obviously has no tests haha

**Comment by dpogue** [comment]
Thanks for finding and fixing this! 🙇🏼 

I see you've updated it for the current version of the docs (12.x), could you also remove it from the dev version of the docs? That way it won't re-appear when we publish a v13. The dev version file should be https://github.com/apache/cordova-docs/blob/master/www/docs/en/dev/core/features/splashscreen/index.md

**Comment by igorsantos07** [comment]
Whoops! Didn't realize that. Done!

File: www/docs/en/12.x/core/features/splashscreen/index.md
Comment (NiklasMerz) [inline www/docs/en/12.x/core/features/splashscreen/index.md:1]: This is still the file in the 12.x directory. I think you need to change that in the `/dev/` directory to get it updated for future versions.

**Review by NiklasMerz — APPROVED** [review]

**Pull request merged**